frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -16,6 +16,7 @@ import { Helmet } from 'react-helmet';
 import { Trans, useTranslation } from 'react-i18next';
 import { Link, match as RouterMatch } from 'react-router-dom';
 import { WatchK8sResource } from '@console/dynamic-plugin-sdk';
+import { ResourceStatus } from '@console/dynamic-plugin-sdk/src/app/components/utils/resource-status';
```

Comment:
Any reason these are not root package exports? You gave one right above this line.

Reply:
Agree with @andrewballantyne here - we should not require consumers of the SDK to dig into the directory structure for components.

Would be nice to see:
```js
import { ResourceStatus, WatchK8sResource } from '@console/dynamic-plugin-sdk';
```

Reply:
@andrewballantyne Added them, thank you